xray/client/index.html

```diff
@@ -18,7 +18,7 @@
    <title>XRay Viewer</title>
 </head>
 <body>
-  <div id="renderArea" style="position: absolute; width: 100%; height: 100%; overflow: hidden;"></div>
+  <div id="renderArea" style="position: absolute; width: 100%; height: 100%; overflow: hidden; touch-action: none;"></div>
```

Comment:
So what was the actual problem?

Reply:
When I zoomed in, the tiles would stay white.

Reply:
I implemented the fix for the error message in search for a fix for the problem, but it did not fix the problem. I still created a PR, but searched further and this fixes the problem.

Reply:
This is Yak shaving depth 3 for me by the way :P

Reply:
Ok, do we also need this fix in https://github.com/googlecartographer/point_cloud_viewer/blob/master/octree_web_viewer/client/index.html#L22?

Reply:
Probably. I added it.

Reply:
Thanks a lot! Can you please also test it?

Reply:
I tried ... I can't see any octree with or without the fix. Can we merge just the xray fix?

Reply:
I just tried your fix, for me the error still occurs.

Reply:
Have you deleted the yarn-generated files and regenerated them?

Reply:
Yes

Reply:
You're right, I have to check again.

Reply:
Is your original approach (https://github.com/cartographer-project/point_cloud_viewer/pull/446/commits/71259c21ea3bb891de936200c1408fdb7e35ba1e) working?

Reply:
#466 fixes the actual problem of tiles staying white...